src/libstd/sys/unix/time.rs

```diff
@@ -86,9 +86,15 @@ mod inner {
     #[link(name = "rt")]
     extern {}
 
+    #[cfg(not(target_os = "netbsd"))]
     extern {
         fn clock_gettime(clk_id: libc::c_int, tp: *mut libc::timespec) -> libc::c_int;
     }
+    #[cfg(target_os = "netbsd")]
+    extern {
+        #[link_name = "__clock_gettime50"]
```

Comment:
These sorts of `#[cfg]` attributes are probably best done via:

```rust
#[cfg_attr(target_os = "netbsd", link_name = "__clock_gettime50")]
```

Could this happen for the modifications to liblibc as well?